site/en/meet-the-team/events/jsconf-chile/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: JSConf Chile
+summary: >-
+  This is the first Chilean edition of JSConf, with two days of international
+  speakers, community, learning and connections.
+image: image/fuiz5I8Iv7bV8YbrK2PKiY3Vask2/9Paz0GyJW9AgEVtmc4yN.jpg
+location: Chile
+date: 2023-02-03
+externalUrl: https://jsconf.cl/
+sessions:
+  - participants:
+      - jakearchibald
+      - adrianajara
+      - petelepage
+      - unakravets
+      - kosamari
+      - jeremywagner
+    description: >-
+      Google community meet up  https://jsconf.cl/events/meet-and-drink with talks by Jeremy,
```

Comment:
This description has a URL in it that isn't linked. If we can't use HTML in here to make it a link maybe remove it?

Reply:
Thanks @rachelandrew . Links aren't supported in the description so I've removed as requested.